diff --git a/Action.c b/Action.c
index f5d7de4..526b769 100644
--- a/Action.c
+++ b/Action.c
@@ -382,7 +382,7 @@ static Htop_Reaction actionRedraw() {
 }
 
 static struct { const char* key; const char* info; } helpLeft[] = {
-   { .key = " Arrows: ", .info = "scroll process list" },
+   { .key = "   hjkl: ", .info = "scroll process list" },
    { .key = " Digits: ", .info = "incremental PID search" },
    { .key = "   F3 /: ", .info = "incremental name search" },
    { .key = "   F4 \\: ",.info = "incremental name filtering" },
@@ -403,7 +403,7 @@ static struct { const char* key; const char* info; } helpRight[] = {
    { .key = "  Space: ", .info = "tag process" },
    { .key = "      c: ", .info = "tag process and its children" },
    { .key = "      U: ", .info = "untag all processes" },
-   { .key = "   F9 k: ", .info = "kill process/tagged processes" },
+   { .key = "   F9 d: ", .info = "kill process/tagged processes" },
    { .key = "   F7 ]: ", .info = "higher priority (root only)" },
    { .key = "   F8 [: ", .info = "lower priority (+ nice)" },
 #if (HAVE_LIBHWLOC || HAVE_LINUX_AFFINITY)
@@ -415,7 +415,7 @@ static struct { const char* key; const char* info; } helpRight[] = {
    { .key = "      s: ", .info = "trace syscalls with strace" },
    { .key = "         ", .info = "" },
    { .key = " F2 C S: ", .info = "setup" },
-   { .key = "   F1 h: ", .info = "show this help screen" },
+   { .key = "   F1 ?: ", .info = "show this help screen" },
    { .key = "  F10 q: ", .info = "quit" },
    { .key = NULL, .info = NULL }
 };
@@ -551,7 +551,7 @@ void Action_setBindings(Htop_Action* keys) {
    keys['q'] = actionQuit;
    keys['a'] = actionSetAffinity;
    keys[KEY_F(9)] = actionKill;
-   keys['k'] = actionKill;
+   keys['d'] = actionKill;
    keys[KEY_RECLICK] = actionExpandOrCollapse;
    keys['+'] = actionExpandOrCollapse;
    keys['='] = actionExpandOrCollapse;
@@ -561,12 +561,11 @@ void Action_setBindings(Htop_Action* keys) {
    keys['S'] = actionSetup;
    keys['C'] = actionSetup;
    keys[KEY_F(2)] = actionSetup;
-   keys['l'] = actionLsof;
+   keys['L'] = actionLsof;
    keys['s'] = actionStrace;
    keys[' '] = actionTag;
    keys['\014'] = actionRedraw; // Ctrl+L
    keys[KEY_F(1)] = actionHelp;
-   keys['h'] = actionHelp;
    keys['?'] = actionHelp;
    keys['U'] = actionUntagAll;
    keys['c'] = actionTagAllChildren;
diff --git a/Panel.c b/Panel.c
index 8c4d44f..c115e27 100644
--- a/Panel.c
+++ b/Panel.c
@@ -375,10 +375,12 @@ bool Panel_onKey(Panel* this, int key) {
    
    int size = Vector_size(this->items);
    switch (key) {
+   case 'j':
    case KEY_DOWN:
    case KEY_CTRL('N'):
       this->selected++;
       break;
+   case 'k':
    case KEY_UP:
    case KEY_CTRL('P'):
       this->selected--;
@@ -393,6 +395,7 @@ bool Panel_onKey(Panel* this, int key) {
       this->selected--;
       break;
    #endif
+   case 'h':
    case KEY_LEFT:
    case KEY_CTRL('B'):
       if (this->scrollH > 0) {
@@ -400,6 +403,7 @@ bool Panel_onKey(Panel* this, int key) {
          this->needsRedraw = true;
       }
       break;
+   case 'l':
    case KEY_RIGHT:
    case KEY_CTRL('F'):
       this->scrollH += CRT_scrollHAmount;
