--- /etc/pacman.conf
+++ /etc/pacman.conf
@@ -27,14 +27,18 @@
 #IgnoreGroup =
 
 #NoUpgrade   =
-#NoExtract   =
+NoExtract   = usr/share/dbus-1/services/*.service
+NoExtract   = etc/profile.d/locale.sh
+NoExtract   = usr/share/vim/vimfiles/archlinux.vim
 
 # Misc options
 #UseSyslog
-#Color
-#TotalDownload
+Color
+TotalDownload
 CheckSpace
-#VerbosePkgLists
+VerbosePkgLists
+ParallelDownloads = 5
+ILoveCandy
 
 # By default, pacman accepts packages signed by keys that its local keyring
 # trusts (see pacman-key and its man page), as well as unsigned packages.
@@ -69,8 +73,12 @@
 # repo name header and Include lines. You can add preferred servers immediately
 # after the header, and they will be used before the default mirrors.
 
-#[testing]
-#Include = /etc/pacman.d/mirrorlist
+[earnestly]
+SigLevel = PackageOptional
+Server = http://teapot.local.lan:8080
+
+[testing]
+Include = /etc/pacman.d/mirrorlist
 
 [core]
 Include = /etc/pacman.d/mirrorlist
@@ -78,8 +86,8 @@
 [extra]
 Include = /etc/pacman.d/mirrorlist
 
-#[community-testing]
-#Include = /etc/pacman.d/mirrorlist
+[community-testing]
+Include = /etc/pacman.d/mirrorlist
 
 [community]
 Include = /etc/pacman.d/mirrorlist
@@ -87,11 +95,11 @@
 # If you want to run 32 bit applications on your x86_64 system,
 # enable the multilib repositories as required here.
 
-#[multilib-testing]
-#Include = /etc/pacman.d/mirrorlist
+[multilib-testing]
+Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
