--- /etc/makepkg.conf	2018-06-29 19:19:02.241627039 +0100
+++ /etc/makepkg.conf	2018-06-29 19:19:54.094691113 +0100
@@ -41,7 +41,7 @@
 CXXFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt"
 LDFLAGS="-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now"
 #-- Make Flags: change this for DistCC/SMP systems
-#MAKEFLAGS="-j2"
+MAKEFLAGS=-j$(nproc)
 #-- Debugging flags
 DEBUG_CFLAGS="-g -fvar-tracking-assignments"
 DEBUG_CXXFLAGS="-g -fvar-tracking-assignments"
@@ -111,9 +111,9 @@
 # Default: put built package and cached source in build directory
 #
 #-- Destination: specify a fixed directory where all packages will be placed
-#PKGDEST=/home/packages
+PKGDEST=$HOME/local/repo
 #-- Source cache: specify a fixed directory where source files will be cached
-#SRCDEST=/home/sources
+SRCDEST=$HOME/local/src
 #-- Source packages: specify a fixed directory where all src packages will be placed
 #SRCPKGDEST=/home/srcpackages
 #-- Log files: specify a fixed directory where all log files will be placed
