diff --git a/etc/pacman.conf b/etc/pacman.conf
index 8ce6919..10bfd87 100644
--- a/etc/pacman.conf
+++ b/etc/pacman.conf
@@ -27,14 +27,15 @@ Architecture = auto
 #IgnoreGroup =
 
 #NoUpgrade   =
-#NoExtract   =
+NoExtract   = usr/share/dbus-1/services/*.service etc/profile.d/locale.sh usr/share/vim/vimfiles/archlinux.vim usr/lib/Xorg.wrap
 
 # Misc options
 #UseSyslog
-#Color
+Color
 #TotalDownload
 CheckSpace
-#VerbosePkgLists
+VerbosePkgLists
+ILoveCandy
 
 # By default, pacman accepts packages signed by keys that its local keyring
 # trusts (see pacman-key and its man page), as well as unsigned packages.
@@ -69,8 +70,12 @@ LocalFileSigLevel = Optional
 # repo name header and Include lines. You can add preferred servers immediately
 # after the header, and they will be used before the default mirrors.
 
-#[testing]
-#Include = /etc/pacman.d/mirrorlist
+[earnestly]
+SigLevel = PackageOptional
+Server = file:///home/earnest/local/repo
+
+[testing]
+Include = /etc/pacman.d/mirrorlist
 
 [core]
 Include = /etc/pacman.d/mirrorlist
@@ -78,8 +83,8 @@ Include = /etc/pacman.d/mirrorlist
 [extra]
 Include = /etc/pacman.d/mirrorlist
 
-#[community-testing]
-#Include = /etc/pacman.d/mirrorlist
+[community-testing]
+Include = /etc/pacman.d/mirrorlist
 
 [community]
 Include = /etc/pacman.d/mirrorlist
@@ -87,11 +92,11 @@ Include = /etc/pacman.d/mirrorlist
 # If you want to run 32 bit applications on your x86_64 system,
 # enable the multilib repositories as required here.
 
-#[multilib-testing]
-#Include = /etc/pacman.d/mirrorlist
+[multilib-testing]
+Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
