diff --git a/link.c b/link.c
index 78dce8a..a8ee909 100644
--- a/link.c
+++ b/link.c
@@ -1958,7 +1958,7 @@ static void * mainloop (void *unused)
                                          strlen (nameddest));
                 obj = pdf_lookup_dest (state.u.pdf, needle);
                 if (obj) {
-                    dest = pdf_parse_link_dest (state.u.pdf, FZ_LINK_GOTO, obj);
+                    dest = pdf_parse_link_dest (state.u.pdf, obj);
 
                     a = desttoanchor (&dest);
                     if (a.n >= 0) {
@@ -2901,7 +2901,7 @@ enum  { uuri, ugoto, utext, uunexpected,
             str_v = caml_copy_string (link->dest.ld.gotor.file_spec);   \
             pageno = link->dest.ld.gotor.page;                          \
             if (pageno == -1) {                                         \
-                gr_v = caml_copy_string (link->dest.ld.gotor.dest);     \
+                gr_v = caml_copy_string (link->dest.ld.uri.uri);        \
                 rty = uremotedest;                                      \
             }                                                           \
             else {                                                      \
@@ -4101,7 +4101,7 @@ static void makestippletex (void)
 
 CAMLprim value ml_fz_version (value unit_v)
 {
-    return caml_copy_string (FZ_VERSION);
+    return caml_copy_string ("1.3");
 }
 
 CAMLprim value ml_init (value pipe_v, value params_v)
