diff --git a/config.ml b/config.ml
index cec0c09..bf957c7 100644
--- a/config.ml
+++ b/config.ml
@@ -7319,11 +7319,11 @@ struct
 let defconfpath =
   let dir =
     try
-      let dir = Filename.concat home ".config" in
-      if Sys.is_directory dir then dir else home
-    with _ -> home
+      Sys.getenv "XDG_CONFIG_HOME"
+    with exn ->
+      Filename.concat home ".config"
   in
-  Filename.concat dir "llpp.conf"
+  Filename.concat dir "llpp/llpp.conf"
 ;;
 
 let confpath = ref defconfpath;;
