diff --git a/config.def.h b/config.def.h
index 9981ca3..886945b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,7 +19,10 @@ enum {
  */
 static const float zoom_levels[] = {
 	 12.5,  25.0,  50.0,  75.0,
-	100.0, 150.0, 200.0, 400.0, 800.0
+	100.0, 150.0, 200.0, 400.0, 800.0,
+	1000.0, 1200.0, 1400.0, 1600.0, 1800,
+	2000.0, 2200.0, 2400.0, 2600.0, 2800,
+	3000.0
 };
 
 /* default slideshow delay (in sec, overwritten via -S option): */
